src/components/organisms/Onboarding/StepHeader.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactElement } from 'react'
+import styles from './StepHeader.module.css'
+
+export default function StepHeader({
+  title,
+  subtitle
+}: {
+  title: string
+  subtitle: string
+}): ReactElement {
+  return (
+    <div className={styles.header}>
+      <h3 className={styles.title}>{title}</h3>
+      <h5 className={styles.subtitle}>{subtitle}</h5>
```

Comment:
question(blocking): I assume `h4` is used for something in-between title and subtitle? If possible we should change this accordingly though. It is not reasonable that the subtitle differs by 2 here.